chatterbox_onnx/opm.py

```
@@ -0,0 +1,43 @@
+        Parameters:
+            sentence (str): Text to synthesize.
+            wav_file (str): Path to the output WAV file that will be written.
+            lang (str, optional): Language override used to select the default voice when no `voice` is provided.
+            voice (str, optional): Voice identifier override to select a specific model.
+        
+        Returns:
+            tuple: (wav_file, phonemes) where `wav_file` is the path to the written WAV file and `phonemes` is `None` when phoneme output is not produced.
+        """
+        if voice is not None and voice != "default" and not os.path.isfile(voice):
+            raise ValueError(
+                "expected a .wav file path for reference voice")  # TODO - consider bundling some defaults and give them names
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)